packages/cross-chain-core/src/providers/wormhole/wormhole.ts

```diff
@@ -290,6 +311,118 @@ export class WormholeProvider
     return { originChainTxnId, destinationChainTxnId };
   }
 
+  async withdraw(
+    input: WormholeWithdrawRequest
+  ): Promise<WormholeWithdrawResponse> {
+    const { sourceChain, wallet, destinationAddress, sponsorAccount } = input;
+    logger.log("sourceChain", sourceChain);
+    logger.log("wallet", wallet);
+    logger.log("destinationAddress", destinationAddress);
+    logger.log("sponsorAccount", sponsorAccount);
```

Comment:
I don't have context, but it seems like this is a lot of logging. Why are we doing so much logging here?